recipe/build.sh

```diff
@@ -127,16 +114,55 @@ if [[ ${HOST} =~ .*linux.* ]]; then
       SKIPS+=(-skip); SKIPS+=(qtlocation)
       SKIPS+=(-skip); SKIPS+=(qt3d)
     fi
+    declare -A COS6_MISSING_DEFINES
+    if [[ ${HOST} == *cos6* ]]; then
+      COS6_MISSING_DEFINES["SYN_DROPPED"]="3"
+      COS6_MISSING_DEFINES["BTN_TRIGGER_HAPPY1"]="0x2c0"
+      COS6_MISSING_DEFINES["BTN_TRIGGER_HAPPY2"]="0x2c1"
+      COS6_MISSING_DEFINES["BTN_TRIGGER_HAPPY3"]="0x2c2"
+      COS6_MISSING_DEFINES["BTN_TRIGGER_HAPPY4"]="0x2c3"
+      COS6_MISSING_DEFINES["BTN_TRIGGER_HAPPY17"]="0x2d0"
+      COS6_MISSING_DEFINES["INPUT_PROP_POINTER"]="0x00"
+      COS6_MISSING_DEFINES["INPUT_PROP_DIRECT"]="0x01"
+      COS6_MISSING_DEFINES["INPUT_PROP_BUTTONPAD"]="0x02"
+      COS6_MISSING_DEFINES["INPUT_PROP_SEMI_MT"]="0x03"
+      COS6_MISSING_DEFINES["INPUT_PROP_MAX"]="0x1f"
+      COS6_MISSING_DEFINES["INPUT_PROP_CNT"]="0x20"
+      COS6_MISSING_DEFINES["ABS_MT_SLOT"]="0x2f"
+      COS6_MISSING_DEFINES["ABS_MT_PRESSURE"]="0x3a"
+      COS6_MISSING_DEFINES["ABS_MT_DISTANCE"]="0x3b"
```

Comment:
These values need to be checked carefully against `/usr/include/linux/input-events-codes.h` from a recent linux distro.

Reply:
While I agree it's good to check and that we should, isn't it the case that the Linux kernel promises not to break userspace. That would be what changing any of these values is to my mind?

Reply:
I meant when reviewing this PR. I might have made typos.